### PR TITLE
Add signing_redirect_url to SignatureRequestRequestOptions

### PR DESCRIPTION
**Pull request by vinassefranche**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
  - As per the api [documentation](https://app.hellosign.com/api/reference#send_signature_request) `signing_redirect_url` is a valid parameter
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by dvdprrsh — CHANGES_REQUESTED** [review]

File: types/hellosign-sdk/index.d.ts [hunk 1]
```diff
@@ -241,6 +241,7 @@ declare namespace HelloSign {
                       | 'YYYY - MM - DD';
               }
             | undefined;
+        signing_redirect_url?: string | undefined;
```
[the remark this answers is not in the document]
Comment (vinassefranche) [reply types/hellosign-sdk/index.d.ts:244]: Nice catch! I've fixed this!

File: types/hellosign-sdk/hellosign-sdk-tests.ts [hunk 2]
```diff
@@ -232,6 +232,7 @@ export const sendSigningRequestWithTemplates = async (pdfFilePath: string) => {
         ],
         template_id: 'test template id',
         metadata,
+        signing_redirect_url: 'http://localhost',
```
[the remark this answers is not in the document]
Comment (vinassefranche) [reply types/hellosign-sdk/hellosign-sdk-tests.ts]: This indeed failed when I fixed the previous comment so I removed it

**Comment by vinassefranche** [comment]
@dvprrsh Is there something wrong with the P.R.?

**Review by dvdprrsh — APPROVED** [review]

**Comment by vinassefranche** [comment]
Ready to merge

**Pull request merged**